src/main/java/com/delo/ak/AkApplication.java

```diff
@@ -0,0 +1,13 @@
+package com.delo.ak;
+
+import org.springframework.boot.SpringApplication;
+import org.springframework.boot.autoconfigure.SpringBootApplication;
+
+@SpringBootApplication
```

Comment:
Do you know what is this (start with @) called in java ? and what is the use of this in general ?

Reply:
The @ symbol in Java is used to indicate annotations. 

@SpringBootApplication
public class AkApplication {

	public static void main(String[] args) {
		SpringApplication.run(AkApplication.class, args);
	}

}

The @SpringBootApplication annotation tells Spring Boot that AkApplication is the main class for the application.